packages/shared/components/popups/ActivityDetailsPopup.svelte

```diff
@@ -1,42 +1,44 @@
 <script lang="typescript">
-    import { Pill, ActivityTypePill, Box, AddressBox, KeyValueBox } from 'shared/components/atoms'
     import { Text, Button } from 'shared/components'
-    import { convertToFiat, currencies, exchangeRates, formatCurrency } from 'shared/lib/currency'
-    import { formatDate, localize } from '@core/i18n'
+    import { localize } from '@core/i18n'
     import { getOfficialExplorer } from 'shared/lib/network'
     import { Platform } from 'shared/lib/platform'
-    import { activeProfile } from 'shared/lib/profile'
-    import { CurrencyTypes } from 'shared/lib/typings/currency'
-    import { Payload } from 'shared/lib/typings/message'
-    import { formatUnitBestMatch } from 'shared/lib/units'
-    import { getMilestoneMessageValue, wallet } from 'shared/lib/wallet'
+    import { getMilestoneMessageValue, isParticipationPayload, wallet } from 'shared/lib/wallet'
     import { FontWeightText } from 'shared/components/Text.svelte'
+    import { TransactionDetails } from 'shared/components/molecules'
     import { getTransactionSubjectAddressOrAccount } from '@lib/utils/transactionObject'
+    import { ActivityType } from '@lib/typings/activity'
 
-    export let id: string
-    export let timestamp: string
-    export let confirmed: boolean
-    export let payload: Payload
-    export let balance: number // migration tx
+    export let message: { id: any; timestamp?: any; confirmed?: any; payload?: any; balance?: any }
```

Comment:
Why use `any` here, if you have all the typings above?

Reply:
The AccountMessage type is not veery friendly because it also has milestone types which we will not display in firefly as far as I am aware... Which means I have to add additional guards in my code when in stardust this object will change anyway. Nevertheless have added correct typing in my next PR:
![Screenshot 2022-05-04 at 12 50 18](https://user-images.githubusercontent.com/89913787/166675551-523324ea-d982-41b1-b302-3136387afce0.png)